.github/workflows/auto_approve_dependency_PRs.yml

```diff
@@ -0,0 +1,23 @@
+name: Auto Approve Dependency MRs
+on:
+  schedule:
+    - cron: '*/10 * * * *'
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Find dependency MRs
```

Comment:
There's also a bunch of `MR` usage in this file that should also get updated to PR. Sorry, forgot to mention that before!

Reply:
Good catch. Fixed.